### ISSUE TITLE
get_listen_now_situations() returns unexpected property 'adTargeting'

### ISSUE THREAD

**Issue by raphaelcastaneda** [simon-weber/gmusicapi#524]
I can repeat this reliably on python 2.7 with the latest develop branch, simply by authenticating a `MobileClient` with a paid account and calling `get_listen_now_situations()`

Error message:

```
gmusicapi.Mobileclient1 (shared:281) [ERROR]: the response format for ListListenNowSituations was not recognized.
```

Request:

```
{u'url': u'https://mclients.googleapis.com/sj/v2.5/listennow/situations', u'headers': {u'Content-Type': u'application/json', u'Authorization': u'<omitted>'}, u'data': '{"requestSignals": {"timeZoneOffsetSecs": -28800}}', u'method': u'POST', u'params': {u'dv': 0, u'tier': u'aa', u'alt': u'json', u'hl': u'en_US'}}

```

Traceback:

```
Traceback (most recent call last):
  File "gmusicapi/protocol/shared.py", line 249, in perform
    cls.validate(response, parsed_response)
  File "gmusicapi/protocol/mobileclient.py", line 627, in validate
    raise_from(ValidationException(str(e)), e)
  File "/usr/local/lib/python2.7/site-packages/six.py", line 718, in raise_from
    raise value
ValidationException: Value {u'kind': u'sj#radioStation', u'byline': u'By Google Play Music', u'name': u'Blogged 50', u'stationSeeds': [{u'kind': u'sj#radioSeed', u'curatedStationId': u'Lj7guwddznce3uza7ox5w5ahruy', u'seedType': u'9'}], u'contentTypes': [u'1'], u'seed': {u'kind': u'sj#radioSeed', u'curatedStationId': u'Lj7guwddznce3uza7ox5w5ahruy', u'seedType': u'9'}, u'skipEventHistory': [], u'compositeArtRefs': [{u'url': u'http://lh3.googleusercontent.com/xhUzRsf-QPaRSRkkJw80w6M6zxc2jIjP-OE5GVnJgZHX-w3LgsyHFzHFzQ', u'kind': u'sj#imageRef', u'aspectRatio': u'2'}, {u'url': u'http://lh3.googleusercontent.com/vaVH1aJ6lLZw9nEUuEp4ag-wSWFZGlVqsSb5TSStwSePYp98FHncaPiN5w', u'kind': u'sj#imageRef', u'aspectRatio': u'1'}], u'adTargeting': {u'keyword': [u'newmusic']}, u'imageUrls': [{u'url': u'http://lh3.googleusercontent.com/cCN7GZ3XW_J9nzY5IW6iMdb2CupMvVMRA82Ehh6SoKDb1bhFPZqlaQfN0eCYgEwBAgElTDvNLw8', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/e2mhkDhr01-Ri_zXDkIOtKEBlGHi9kOvuXXyKqo9br1zTNonOgM2GUL87MzWE8e1XB6TuOcF', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/DLqPkch_oROwV6oN_ycaF3KHzPP5w3mVP4poa0YUafS4N7uGo9dGgLHZ25ehq2BouCC4mLNwuQ', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/bPS7Iv67_xMx07eezkBJSEciCpvyT7N-4CZiAobTfsE4F-PTA8xJ2aM8hivFPWXIrZ4KMdxUJYk', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/Ey-CJqbSZ06PVE6KnRbyk3tcrl2Z6ykm0yivBD68N0pcxLxDOjyG6_0NofEC9-aP5VJ8UZqvpg', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/_RzvKgJM7Hur4B8DcdKzBbPhn27QM7hqTri6HZF0DK4-3ra-nUnAqNmiT85kZWYpLhNcQOzhAA', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/eqUY_xc8J3cpqd1OPuBv-grNfbRwM1Sf51wpvJQqGwAhD9A2R262qqAWYM5U_Fgh9f9Akm9uHw', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}, {u'url': u'http://lh3.googleusercontent.com/8VuoQXzu8ff37eH_xdfoJ9Znyy-r6z0AYBA2Vy8J7y_UmX0xI7RsC_nI89gjWk1LTf11hjlFS8Q', u'kind': u'sj#imageRef', u'aspectRatio': u'1', u'autogen': False}], u'description': u'A curated selection of the best songs trending on the Internet right now.'} for field '<obj>.situations[1].stations[2]' contains additional property 'adTargeting' not defined by 'properties' or 'patternProperties' and additionalProperties  is False
```

**Comment by simon-weber** [comment]
Cool, thanks for letting us know. Should just be a quick schema change.

**Comment by simon-weber** [comment]
I fixed this in 04900a8d and forgot to close the issue.
